source/1_engine/rex_engine/src/memory/memory_tracking.cpp

```diff
@@ -146,7 +152,13 @@ namespace rex
 
   void MemoryTracker::track_dealloc(MemoryHeader* header)
   {
-    REX_WARN_X(LogEngine, header->frame_index() != globals::frame_info().index(), "Memory freed in the same frame it's allocated (please use single frame allocator for this)");
+    // This is possible if static data gets deleted after the memory tracker is already destructed
+    if(!m_is_active)
+    {
+      return;
+    }
+
+    // REX_WARN_X(LogEngine, header->frame_index() != globals::frame_info().index(), "Memory freed in the same frame it's allocated (please use single frame allocator for this)");
```

Comment:
Is there an issue for creating this frame allocator?

Reply:
No, the frame allocator won't call into the track_dealloc code at the end of every frame

Reply:
Then how do you plan on resolving this?